lib/std/math/big/int.zig

```diff
@@ -1959,6 +1959,45 @@ pub const Const = struct {
         return bits;
     }
 
+    /// @popCount with two's complement semantics.
```

Comment:
popcount is only defined for unsigned integers, so it would be more clear to be explicit here with something like "the number of  bits with value 1 (including the sign bit)" are meant. https://gcc.gnu.org/onlinedocs/gccint/Integer-library-routines.html#Integer-library-routines
```
Runtime Function: int __popcountsi2 (unsigned int a)
Runtime Function: int __popcountdi2 (unsigned long a)
Runtime Function: int __popcountti2 (unsigned long long a)

    These functions return the number of bits set in a. 
```

Reply:
I think the usage of the term "two's complement semantics" is more or less on par with the other doc comments in the file, including the other `popCount`s, but it could only help to be clearer. I can add a note about inclusion of the sign bit.